src/m365/spo/commands/hubsite/hubsite-get.ts

```diff
@@ -39,8 +66,26 @@ class SpoHubSiteGetCommand extends SpoCommand {
 
         return request.get(requestOptions);
       })
-      .then((res: any): void => {
-        logger.log(res);
+      .then((res: HubSite): Promise<CommandOutput | void> => {
+        hubSite = res;
+
+        if (args.options.includeAssociatedSites !== true || args.options.output && args.options.output !== 'json') {
+          return Promise.resolve();
+        }
```

Comment:
TBH I think that's good approach to just resolve here. By doing that we will not stop the command from moving forward but instead we will just go on and resolve in error on line 84 as the associatedSitesCommandOutput will be undefined. 
Instead I suggest we throw error here like 
`throw Error(`includeAssociatedSites option is only allowed with json output mode);`
That way when user will do something wrong the output will be
![image](https://user-images.githubusercontent.com/58668583/178069889-a9e8e758-c0c5-4f7d-b4eb-b8077aee89ef.png)

now it would just break like 
![image](https://user-images.githubusercontent.com/58668583/178069931-cbd66aa6-5b12-4ac0-a18b-400153f1d25c.png)